test/dfusionMultiSafes.js

```diff
@@ -165,52 +167,136 @@ contract("GnosisSafe", function(accounts) {
       }
     })
 
-    it("transfers tokens from fund account through trader accounts and into exchange via automatic deposit logic", async () => {
+    it("transfers tokens from fund account through trader accounts and into exchange via automatic deposit logic, p> 1", async () => {
```

Comment:
This test creates orders, but order creation is tested only after this test. Maybe we should move this test after the one testing orders and call it "test deposit creation from orders, p>1"? Same reasoning for the next test, p<1.

Reply:
I think that we are checking in this test the transfer amounts and not the order creation. Hence I would keep it in the "  describe("transfer tests:", async function() { " block.
Or would you create another describe block like this:
 "  describe("transfer tests depended on order information:", async function() { "

Reply:
Yeah, I was thinking to have a new block since this is not a "pure" deposit. But it's also fine as it is now.